File: C++/day5.cpp
#include <cmath>
#include <cstdio>
#include <vector>
#include <iostream>
#include <algorithm>
using namespace std;


int main() {
    /* Enter your code here. Read input from STDIN. Print output to STDOUT */
    int T,a,b,N,sum = 0;
    cin>>T;
    for(int i=0;i<T;i++)
    {
        cin>>a>>b>>N;
        sum = a+b;
        cout<<sum<<" ";
        for(int j=1;j<N;j++)
        {
            sum = sum + (pow(2,j) * b);
            cout<<sum<<" ";
        }
        cout<<endl;
    }
    return 0;
}

File: JAVASCRIPT/day5.js
function processData(input) {
    //Enter your code here
    var data = input.split('\n');
    var i;

    for (i = 0; i < Number(data[0]); i++) {
        var input = data[i + 1].split(' ');
        var a = Number(input[0]);
        var b = Number(input[1]);
        var N = Number(input[2]);
        var initial = a + Math.pow(2,0) * b;
        var series = [initial];

        for (var j = 1; j < N; j++) {
            initial += Math.pow(2,j) * b;
            series.push(initial);
        }

        console.log(series.join(' '));
    }
}